contracts/libs/CoreRefPauseableLib.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import {CoreRef} from "../refs/CoreRef.sol";
+import {Pausable} from "@openzeppelin/contracts/security/Pausable.sol";
+
+/// @title PauseableLib
+/// @notice PauseableLib is a library that can be used to pause and unpause contracts, among other utilities.
+/// @dev This library should only be used on contracts that implement CoreRef.
+library CoreRefPauseableLib {
```

Comment:
Personally think it's cleaner to just cast the coreRef address to core and call pause than use a library for it. No strong feelings